src/__stories__/centered-carousel-story.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import {
+    CenteredCarousel,
+    PageBullets,
+    Stack,
+    Box,
+    Circle,
+    ResponsiveLayout,
+    useTheme,
+    Text2,
+    Callout,
+    IconInformationRegular,
+    Text10,
+    ThemeVariant,
+    SectionTitle,
+} from '..';
+
+export default {
+    title: 'Components/Carousel/CenteredCarousel',
+};
+
+const centerStyle = {display: 'flex', alignItems: 'center', justifyContent: 'center'};
+
+type Args = {numItems: number};
+
+export const Default: StoryComponent<Args> = ({numItems}) => {
+    const [pageInfo, setPageInfo] = React.useState<{
+        pageIndex: number;
+        shownItemIndexes: Array<number>;
+    } | null>(null);
+    const {colors} = useTheme();
+    return (
+        <Box paddingY={24}>
+            <ResponsiveLayout>
+                <Stack space={16}>
+                    <Callout
+                        description="Resize the window to see centerd carousel style in mobile. Arrow controls disappear in touch devices."
+                        icon={<IconInformationRegular />}
+                    />
+                    <CenteredCarousel
+                        onPageChange={setPageInfo}
+                        renderBullets={(bulletsProps) => (
```

Comment:
bullets behavior it's quite weird in centered carousel. I understand that bullets always shows pages but in this case looks like a bug

Reply:
It's not only in the centered carousel. It happens in the normal carousel too. That is what happen when `itemsToScroll` < `itemsPerPage`. As bullets represent pages, they only change when a full page is changed, that causes a weird behavior when you click the next button and the carousel advances one item but the bullets don't change.

I don't know what's the best solution for this. But I'd try to always use an `itemsPerPage` === `itemsToScroll` to avoid this.